clickstream/docker-compose-automatic.yml

```diff
@@ -0,0 +1,209 @@
+---
+version: '2'
+services:
+  zookeeper:
```

Comment:
This is where you could add container names so it shows up better for referencing in `docker-compose exec`, etc

Reply:
thanks, I'll do that in the future

Reply:
Done: https://github.com/confluentinc/examples/pull/84/commits/22410d3864eaf7609ccefb181cf15af640243f8a